ConfigurationSystem/Agent/RucioSynchronizer.py

```diff
@@ -0,0 +1,359 @@
+            newRSE = False
+            self.log.info("Synchronizing SEs")
+            result = getStorageElements()
+            if result['OK']:
+                rses = [rse['rse'] for rse in client.list_rses()]
+                for se in result['Value']:
+                    if se not in rses:
+                        # The SE doesn't exist. Will create it
+                        newRSE = True
+                        self.log.info('%s does not exist and will be created' % se)
+                        try:
+                            client.add_rse(rse=se, deterministic=True, volatile=False)
+                        except Exception as err:
+                            self.log.error('Cannot create RSE' % str(err))
+                            continue
+
+                        # Add RSE attributes for the new RSE
+                        dictRSEAttributes = {'naming_convention': 'BelleII', 'ANY': True, 'fts': 'https://fts.usatlas.bnl.gov:8446'}
+                        dictRSEAttributes = {'naming_convention': 'BelleII', 'ANY': True, 'fts': 'https://kek2-fts01.cc.kek.jp:8446'}
+                        for key in dictRSEAttributes:
+                            self.log.info('On %s, setting %s : %s' % (se, key, dictRSEAttributes[key]))
+                            client.add_rse_attribute(se, key, value=dictRSEAttributes[key])
+                        client.set_local_account_limit('root', se, 100000000000000000)
+                        if se.find('TMP-SE') > -1:
+                            client.add_rse_attribute(se, 'TMP-SE', True)
+                            client.set_local_account_limit(account, se, 100000000000000000)
+
+                    # Create the protocols
+                    try:
+                        protocols = client.get_protocols(se)
+                    except RSEProtocolNotSupported as err:
+                        self.log.info('Cannot get protocols for %s : %s' % (se, err))
+                        protocols = []
+                    existing_protocols = []
+                    for prot in protocols:
+                        existing_protocols.append((str(prot['scheme']), str(prot['hostname']), str(prot['port']), str(prot['prefix'])))
+                    protocols_to_create = []
+                    for cnt in range(len(result['Value'][se])):
+                        params = result['Value'][se][cnt]
+                        prot = (str(params['scheme']), str(params['hostname']), str(params['port']), str(params['prefix']))
+                        protocols_to_create.append(prot)
+                        if prot not in existing_protocols and prot[0] in valid_protocols:
+                            # The protocol defined in Dirac does not exist in Rucio. Will be created
+                            self.log.info('Will create new protocol %s://%s:%s%s on %s' % (params['scheme'], params['hostname'], params['port'], params['prefix'], se))
```

Comment:
You should run the linting from one of the newer branches, as a few lines are far too long

Reply:
Done